### ISSUE TITLE
Woocommerce Single Product Content Widget is rendered only once

### ISSUE THREAD

**Issue by mmai1554** [elementor/elementor#19946]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

On an Elementor single product template we need the product content widget 2 times. One for desktop view and one for mobile view since the layout is somewhat more complex. It turns out, the product content widget is rendererd only the first time on the page but not the second time.

### Steps to reproduce

1. create product page template
2. use product content widget
3. use another product content widget
4. both widgets are displayed in the elementor editor. 
5. save the template
6. on the frontend only the first widget is rendered.

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

					
== Server Environment ==
	Operating System: Darwin
	Software: Apache/2.4.46 (Unix) OpenSSL/1.0.2u mod_fastcgi/mod_fastcgi-SNAP-0910052141 mod_wsgi/3.5 Python/2.7.18 mod_perl/2.0.11 Perl/v5.30.1
	MySQL version: MySQL Community Server (GPL) v5.7.34
	PHP Version: 8.1.1
	PHP Memory Limit: 1024M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 8M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.0.2
	Site URL: https://play-elementor.test
	Home URL: https://play-elementor.test
	WP Multisite: No
	Max Upload Size: 8 MB
	Memory limit: 40M
	Max Memory limit: 1024M
	Permalink Structure: /%year%/%monthnum%/%postname%/
	Language: de-DE
	Timezone: 0
	Admin Email: info@mainetcare.com
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.6.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: de_DE
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:105.0) Gecko/20100101 Firefox/105.0

== Active Plugins ==
	Elementor
		Version: 3.7.7
		Author: Elementor.com

	Elementor Pro
		Version: 3.7.7
		Author: Elementor.com

	WooCommerce
		Version: 6.8.2
		Author: Automattic

== Elemente Verwendung ==
	
	product : 1
		heading : 1
		woocommerce-product-content : 2

== Elementor-Experimente ==
	Optimierte DOM Ausgabe: Standardmäßig aktiviert
	Verbessertes Laden von Assets: Standardmäßig aktiviert
	Verbessertes Laden von CSS: Standardmäßig aktiviert
	Inline-Schriftarten-Symbole: Standardmäßig deaktiviert
	Verbesserungen der Zugänglichkeit: Standardmäßig aktiviert
	Zusätzliche benutzerdefinierte Breakpoints: Standardmäßig aktiviert
	Vorlagen Kit importieren / exportieren: Standardmäßig aktiviert
	Native WordPress-Widgets aus den Suchergebnissen ausblenden: Standardmäßig aktiviert
	admin_menu_rearrangement: Standardmäßig deaktiviert
	Flexbox Container: Standardmäßig deaktiviert
	Default to New Theme Builder: Standardmäßig aktiviert
	Hello Theme Header & Footer: Standardmäßig aktiviert
	Startseiten: Standardmäßig aktiviert
	Farbpalette: Standardmäßig aktiviert
	Bevorzugte Widgets: Standardmäßig aktiviert
	Admin Leiste: Standardmäßig aktiviert
	Page Transitions: Standardmäßig aktiviert
	Notes: Standardmäßig aktiviert
	Form Submissions: Standardmäßig aktiviert
	Scroll Snap: Standardmäßig aktiviert

== Protokoll ==
	
PHP: showing 3 of 3PHP: 2022-09-28 17:26:02 [warning X 2][/Users/mmai/www/wordpress/play-elementor/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-09-28 17:27:01 [notice X 5][/Users/mmai/www/wordpress/play-elementor/wp-content/plugins/elementor-pro/core/utils/collection.php::75] Return type of ElementorPro\Core\Utils\Collection::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-09-28 17:27:26 [notice X 7][/Users/mmai/www/wordpress/play-elementor/wp-content/plugins/elementor-pro/core/database/model-base.php::135] Return type of ElementorPro\Core\Database\Model_Base::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: /Users/mmai/www/wordpress/play-elementor/wp-content/plugins/elementor-pro/core/database/model-base.php(10): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /Users/mmai/www/wordpress/play-elementor/wp-content/plugins/elementor-pro/plugin.php(153): class type include()array (
  0 => '/Users/mmai/www/wordpress/play-elementor/wp-content/plugins/elementor-pro/core/database/model-base.php',
)
#2: /Users/mmai/www/wordpress/play-elementor/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php(18): ElementorPro\Plugin -> autoload()
#3: /Users/mmai/www/wordpress/play-elementor/wp-content/plugins/elementor-pro/plugin.php(153): class type include()array (
  0 => '/Users/mmai/www/wordpress/play-elementor/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php',
)
#4: /Users/mmai/www/wordpress/play-elementor/wp-content/plugins/elementor-pro/modules/notes/data/controller.php(236): ElementorPro\Plugin -> autoload()
',
)]

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Kompatibel

== Elementor Pro - Compatibility Tag ==

**Comment by nicholaszein** [comment]
Hello @mmai1554,

Thank you for reaching out! 🙏 

#### ⛔ While reviewing your report we concluded this is not an issue caused by Elementor. Here's what we found out:

TL;DR: You can't have more than one Post Content or Product Content widget in your Single Templates. Even if one is hidden (it's cosmetical) in some responsive modes.

The reason: The Product Content widget uses the same principles of WordPress to show any post-type content, which is to call the `the_content()` function to show the post-type content. By default WordPress will _**only output the first instance of this function once**_. This is by design to avoid unwanted duplicates or infinite loops that might occur when more than one instance of this function is loaded. Elementor respects, and follows the same WordPress standards. 

You can refer to [this article](https://elementor.com/help/post-content-widget/) in the Elementor Help center. Note that the Post Content and Product Content widgets do exactly the same thing, and work in exactly the same way. The difference is that products have more information that isn't loaded inside `the_content`, and by default the actual `the_content` loads inside the product tabs, and is labeled as "Product Description" inside WooCommerce. But Elementor offers the option to have the Product Description somewhere else using the Product Content widget, which is something you can't have by default in WooCommerce.

### ✅ What you can do to solve your issue:
Use only one instance of the Post/Product Content widget inside your Single templates. Find another way to relocate the position of the widget. A solution to this would be to use custom CSS, more specifically CSS Grids if you really need to change the position of this widget on different screens.

I hope this helps. 🙂 

For now, that's all we can help with. If you think this is still an issue caused by Elementor, please **feel free to open a [new bug report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml)** and provide us with the most information you can possibly get by following our guidelines:

* Please create GitHub issues **only for Elementor bugs**. GitHub issues ARE NOT FOR SUPPORT or for issues with 3rd party plugins.
* To **avoid duplicates**, please search this repository for similar issues before submitting your own.
* If the issue can't be recreated on other sites, it's not an Elementor bug. In that case, **use the support links below before submitting your issue here**.
* Make sure you are using the most **updated versions** of Elementor, WordPress & your theme.
* Deactivate all of your plugins. If this solves the problem, gradually activate your plugins one by one, until you spot the problematic plugin, and then **contact the 3rd party plugin developer**.
* Make sure to **provide a detailed description** of your issue.
* Make sure to **provide detailed steps to replicate** your issue.
* Make sure the issue occurs with **only Elementor** (and Elementor Pro) plugin(s) active.
* Make sure the issue happens with a **blank theme** active ([Hello Elementor](https://wordpress.org/themes/hello-elementor/)).
* Make sure to **provide a copy of your _Elementor > System Info_**. Download the `.txt` file and drag it inside the "System Info" field when submitting a new bug report.

❓ If you have questions or need general support, please use one of the following channels:

* [Elementor Community](https://facebook.com/groups/Elementors/) on Facebook
* [Elementor Knowledge base](https://docs.elementor.com)
* [Support forums](https://wordpress.org/support/plugin/elementor) on WordPress.org 
* [Elementor Developers docs](https://developers.elementor.com/)

💎 If you have purchased [Elementor Pro](https://elementor.com/pro), you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.

Kind regards

**Comment by nicholaszein** [comment]
Hello @mmai1554!

Thanks for the feedback! I've already sent word for the people responsible to improve the documentation. 😉 

Best regards
